.travis.yml

```diff
@@ -1,130 +0,0 @@
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.6"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-  allow_failures:
-    - python: "3.9"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.9"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.9"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.8"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.8"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.8"
-      env: DJANGO="https://github.com/django/django/archive/main.tar.gz" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.7"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.7"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.7"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.6"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.6"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.6"
-      env: DJANGO="Django==2.1.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.7"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=postgres
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=mysql-innodb
-    - python: "3.6"
-      env: DJANGO="Django==2.0.*" IMPORT_EXPORT_TEST_TYPE=sqlite
-
-install:
-  - pip install -q $DJANGO
-  - pip install -r requirements/base.txt
-  - pip install -r requirements/test.txt
-  - pip install coveralls
-script:
-   - PYTHONPATH=".:tests:$PYTHONPATH" python -Wall -m coverage run --omit='setup.py' --source=. tests/manage.py test core --settings=
-   - if python -c 'import sys; sys.exit(1 - (sys.version_info >= (3, 6)))'; then isort --check-only import_export tests; fi
```

Comment:
I can't find the isort test on the github workflow, not a showstopper though.

Reply:
Hey Manel - thanks for reviewing.  I'll take a look into isort

Reply:
Have added isort checks